src/controllers/ProductsController.js

```diff
@@ -415,7 +415,6 @@ class ProductsController extends Controller {
 
     const devices = (await this._deviceAttributeRepository.getManyFromIDs(
       deviceIDs,
-      this.user.id,
```

Comment:
That's true for now but this will be a bug later if we open up this API to more than the admin account.  I would think that the permissions check would still allow admin accounts to access other devices.